guides/doc-Release_Notes/topics/katello-4.14.0.adoc

```diff
@@ -0,0 +1,127 @@
+* pass:[sudo katello-tracer-upload fails] - https://projects.theforeman.org/issues/37561[#37561]
+* pass:[Host and hostgroup form shows multiple Library entries] - https://projects.theforeman.org/issues/37174[#37174]
+
+=== API
+
+* pass:[Add Hammer & API support for host multi-CV] - https://projects.theforeman.org/issues/37669[#37669]
+
+=== Activation Key
+
+* pass:[AKs description of registration_commands needs more info] - https://projects.theforeman.org/issues/37559[#37559]
+
+=== Alternate Content Sources
+
+* pass:[ACS - throw proper errors for ULN ACS URLs] - https://projects.theforeman.org/issues/35582[#35582]
+
+=== Container
+
+* pass:[Limit v1 search to v1 container clients on Capsule] - https://projects.theforeman.org/issues/37705[#37705]
+* pass:[Create Katello push repositories as needed at container push time] - https://projects.theforeman.org/issues/37455[#37455]
+
+=== Content Views
+
+* pass:[Newly published CV version shows need_published as true] - https://projects.theforeman.org/issues/37633[#37633]
+* pass:[katello_repository_debs "id" column hits max integer size] - https://projects.theforeman.org/issues/37585[#37585]
+* pass:[Content import task failed indexing errata from Pulp] - https://projects.theforeman.org/issues/37549[#37549]
+* pass:[CV promote fails with undefined method `get_status' for nil:NilClass when deleting a Host in the CV during Finalize phase of the Promote task] - https://projects.theforeman.org/issues/37543[#37543]
+* pass:[Removing CV error's with "Cannot delete record because of dependent content_facet" ] - https://projects.theforeman.org/issues/37538[#37538]
+* pass:[Very long CV names/labels display weirdly on CV UI] - https://projects.theforeman.org/issues/37530[#37530]
+
+=== Foreman Proxy Content
+
+* pass:[[DEV] [RFE] Need an option to turn off the 'Reclaim Space' warning] - https://projects.theforeman.org/issues/37716[#37716]
+* pass:[Slow smart proxy sync in 4.11] - https://projects.theforeman.org/issues/37356[#37356]
+
+=== Host Collections
+
+* pass:[Host Collections widget should respect the errata_status_installable setting] - https://projects.theforeman.org/issues/37288[#37288]
+
+=== Hosts
+
+* pass:[Content source is changed to wrong proxy if you simply press the submit button on the host page] - https://projects.theforeman.org/issues/37709[#37709]
+* pass:[Host update failure: param is missing or the value is empty: content_facet_attributes] - https://projects.theforeman.org/issues/37704[#37704]
+* pass:[Convert2rhel env facts are getting filtered due to env ethernet regex] - https://projects.theforeman.org/issues/37696[#37696]
+* pass:['TypeError: getCustomizedRexUrl is not a function' when you go to Review step in Packages wizard] - https://projects.theforeman.org/issues/37684[#37684]
+* pass:[Add postgresql-evr as a migration] - https://projects.theforeman.org/issues/37678[#37678]
```

Comment:
I think this was reverted and shouldn't be in the changelog. I reopened the issue.